projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -260,6 +261,9 @@ class MyPlanHeader extends React.Component {
 						</div>
 					</Card>
 				) }
+				<Card compact>
+					<License />
+				</Card>
```

Comment:
Should we show this to all Jetpack site owners? Given that only a subset of sites will be able to use licenses, it may be confusing if we add this input field here. It may generate support requests of folks asking for a license when they cannot actually get one.

Would it be an option to only display the field when a filter is set on the site, and that filter would then be set in an mu-plugin by a partner supporting licensing, kind of like we did with the Wizard setup?

Reply:
I added the code to only display this part of the UI if the filter `jetpack_show_licensing_ui` is `true`. I'd like to get feedback on my approach here.

Reply:
Yeah, I think that works well that way! 👍